pkg/cluster/cluster.go

```diff
@@ -30,10 +30,6 @@ import (
 	"k8s.io/kubectl/pkg/scheme"
 )
 
-var (
-	DefaultScript = "http://rancher-mirror.cnrancher.com/k3s/k3s-install.sh"
```

Comment:
If the user didn't set `--k3s-install-script`, we could use this value as the default value in each provider.

Reply:
Already set a default value on each provider